src/extension/background-script/actions/swaps/create.ts

```diff
@@ -0,0 +1,24 @@
+import state from "~/extension/background-script/state";
+import { MessageCreateSwap } from "~/types";
+
+const createSwap = async (message: MessageCreateSwap) => {
+  try {
+    const connector = await state.getState().getConnector();
```

Comment:
we need it like this now,
but I am wondering if this should be a function on the connector. We move in a direction to add more specialized functions to the connector interface that are actually not universal. 

I think we should try to move this out of the connector and onto the action level here at some point. 
this would make it also easier to have configurable swap services. Those then also just use the connector do pay/create invoices, etc.

Reply:
It was like this originally and we did a check if the connector type was Alby. Is there a better way?

Reply:
I think for now not. but maybe at some point we support others there and it's open.